docs/general/polkadot-direction.md

```diff
@@ -0,0 +1,265 @@
+
+The slot auction mechanism is not agile, creates high entry barriers, and is designed for
+long-running single applications (i.e., the original Polkadot vision proposed in the whitepaper).
+
+We depart from the classic lease auctions and propose an agile marketplace for coretime, where
+essentially **coretime becomes a commodity that can be tokenized, sold and traded**. This setup
+maximizes the agility of Polkadot and lets the market figure out the best solution needed for
+applications to be successful.
+
+Applications will be able to reserve **bulk coretime** and **instantaneous coretime** depending on
+their needs. Bulk coretime rental will be a standard rental of coretime through a broker system
+parachain at fixed price for a fixed period of time. Instantaneous coretime rental will be available
+through ongoing sale of coretime for immediate use at a spot price. This system will lowers the
+barrier-to-entry for prospective builders.
+
+Revenues from coretime sales can for example be burnt, used to fund the Treasury, or used for a mix
+of those options. The topic is currently under discussion. For more information, see
+[RFC-0010](https://github.com/polkadot-fellows/RFCs/pull/10) and
+[RFC-0015](https://github.com/polkadot-fellows/RFCs/pull/17/files).
+
+## From Chain- to Application-centricity
+
+Polkadot 1.0 was a chain-centric paradigm consisting of isolated chains able to exchange messages.
+This was not fundamentally different from having completely different chains connected to bridges,
+with the only difference of having the relay-chain securing the network, providing message-passing
+capability, and doing some extra tasks such as [crowdloans](../learn/learn-crowdloans.md),
+[auctions](../learn/learn-auction.md), [staking](../learn/learn-staking.md),
+[accounts](./learn-accounts-index), [balances](../learn/learn-balance-transfers.md), and
+[governance](../learn/learn-polkadot-opengov.md). Having a chain-centric system will ultimately end
+in chain-centric application and UX.
+
+The true innovation of Polkadot is about leveraging the unique value proposition offered by
+different chains and using those chains’ collaborative potential to build inter-chain applications
+to solve real-world problems. Those applications will thus need to span across chains.
+
+**Increasingly fewer tasks will be handled by the relay-chain** that will focus efforts only on
+primary tasks: securing the network and providing secure message-passing capability.
+[System parachains](../learn/learn-system-chains.md) will be used to take over secondary relay-chain
+tasks such as staking, governance, etc.
+
+### XCM and Accords
+
+[XCM](../learn/learn-xcm-transport.md) is a format, a language of intention abstract over
+functionality common within chains. It creates an expressive language of what you intend to do or
+want to happen. There are no guarantees that chains faithfully interpret XCM messages; thus, XCM is
```

Comment:
i think this needs to be slightly re-worded.

as someone reading this, if XCM isnt good for trustless environments, and chains my not faithfully interpret XCM messages, why do we have it at all?

I think you need to preface the different actors and needs those actors might have, and the goal of providing different options for developers to use for their needs and designs